examples/index.json

```diff
@@ -46,6 +46,13 @@
                 "verifyBuild": true,
                 "verifyOutput": true,
                 "isLearnByExample": true
+            },
+            {
+                "name": "Expression-oriented style",
```

Comment:
Is this in the correct category? IMO, it should go under the "Functions" category or have its own category.

Reply:
Probably the latter since in the slide we talk about more than just expression-bodied functions.